src/service/navigation.js

```diff
@@ -231,14 +235,18 @@ export class Navigation {
 
     const location = this.parseUrl_(target.href);
 
-    // Handle AMP-to-AMP navigation if rel=amphtml.
-    if (this.handleA2AClick_(e, target, location)) {
-      return;
-    }
+    // Ignore handling A2A and custom protocol if it is invoked via a right
+    // click. See github issue #14768.
+    if (!isRightClick) {
```

Comment:
Instead, we should probably just early-out after `urlReplacements.maybeExpandLink(...)`. `handleNavClick_()` isn't useful for right-click, right?

Actually it might be more readable to have a separate handler method for left and right click, and move shared logic to a new function that they both can call.